docs/sources/panels/configure-panels/index.md

```diff
@@ -0,0 +1,111 @@
+---
+aliases:
+  - /docs/grafana/latest/panels/add-panels-dynamically/about-repeating-panels-rows/
+  - /docs/grafana/latest/panels/add-panels-dynamically/configure-repeating-rows/
+  - /docs/grafana/latest/panels/add-panels-dynamically/configure-repeating-panels/
+  - /docs/grafana/latest/panels/add-panels-dynamically/
+  - /docs/grafana/latest/panels/repeat-panels-or-rows/
+  - /docs/grafana/latest/panels/working-with-panels/add-title-and-description/
+  - /docs/grafana/latest/panels/working-with-panels/view-json-model/
+  - /docs/grafana/latest/panels/configure-panels/
+title: Configure panels
+menuTitle: Configure panels
+weight: 150
+keywords:
+  - panel
+  - dynamic
+  - rows
+  - add
+  - title
+  - description
+  - JSON model
+---
+
+# Configure panels
+
+You can configure Grafana to dynamically add panels or rows to a dashboard. A dynamic panel (or row) is a panel that the system creates based on the value of a variable. Variables dynamically change your queries across all panels in a dashboard.
+
+You can see examples in the following dashboards:
+
+- [Prometheus repeat](https://play.grafana.org/d/000000036/prometheus-repeat)
+- [Repeated Rows Dashboard](https://play.grafana.org/d/000000153/repeat-rows)
```

Comment:
feels very wrong to start with this advanced topic. I would start with the basics of adding panels, going into panel edit, changing panel title / description. 

An introduction to panel editor UI etc. 

Repeating panels and rows is an advanced feature I would have at the bottom

Reply:
Maybe start with something like:

Dashboards are the main building blocks of a dashboard. You add panels via the add panel button located in the top toolbar. 

To edit a panel you have to open the panel editor. Click anywhere in the panel title to open the panel menu, then select the `Edit` option. You can also use the keyboard shortcut `e` while hovering over the panel you want to edit. 

This will open the panel editor. Here you can add and edit queries, data transformations, and panel and visualization settings.

Reply:
Sure, I can add that content in - other than adding the intro and edit, what other tasks would you consider to be configuring panels?